great_expectations/validator/validator.py

```diff
@@ -1028,6 +1028,7 @@ def graph_validate(
                 expectation_validation_graphs=expectation_validation_graphs,
                 evrs=evrs,
                 processed_configurations=processed_configurations,
+                show_progress_bars=self._determine_progress_bars(),
```

Comment:
This is a great catch @tommy-watts-depop thank you for the contribution. LGTM

Reply:
Looks like the integration tests are failing?